client/informant/src/lib.rs

```diff
@@ -29,7 +29,6 @@ use sp_runtime::traits::{Block as BlockT, Header};
 use sp_transaction_pool::TransactionPool;
 use sp_utils::{status_sinks, mpsc::tracing_unbounded};
 use std::{fmt::Display, sync::Arc, time::Duration, collections::VecDeque};
-use parking_lot::Mutex;
```

Comment:
As far as I can tell `parking_lot` is not used in `informant` after this change.

```diff
diff --git a/Cargo.lock b/Cargo.lock
index 8a84eaf3b..0b816c6a8 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -6420,7 +6420,6 @@ dependencies = [
  "futures 0.3.4",
  "log",
  "parity-util-mem",
- "parking_lot 0.10.2",
  "sc-client-api",
  "sc-network",
  "sp-blockchain",
diff --git a/client/informant/Cargo.toml b/client/informant/Cargo.toml
index d2df78537..9d2639123 100644
--- a/client/informant/Cargo.toml
+++ b/client/informant/Cargo.toml
@@ -23,4 +23,3 @@ sp-blockchain = { version = "2.0.0-rc4", path = "../../primitives/blockchain" }
 sp-runtime = { version = "2.0.0-rc4", path = "../../primitives/runtime" }
 sp-utils = { version = "2.0.0-rc2", path = "../../primitives/utils" }
 sp-transaction-pool = { version = "2.0.0-rc2", path = "../../primitives/transaction-pool" }
-parking_lot = "0.10.2"
(END)
```